.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 Cargo.lock
 .env
+.idea
```

Comment:
Please add this in your git global ignore instead, thanks!

Reply:
thank you @Xuanwo, updated it, please take a look again.